### DISCUSSION TITLE
Path not exist!!

### DISCUSSION THREAD

**Discussion by amit071997** [StackStorm/st2#6180] [Q&A]
Hi All
I have written a custom script for applying NTFS permissions on a shared path. Locally the script is running fine but when i am trying to run through stackstorm and passing the path in env as an object it shows me an error **Bad unicode escape in JSON**

I have tried using double backslash, the error got removed but when the action got run it throws an error that path didn't exist.
have also used forward slash, but unfortunately these workarounds didn't work.

Any suggestion will be appreciated.

Thanks .

if you need more information about this query please comment on this.

**Comment by chain312** [comment]
I think you should give more error information.

**Comment by docbyte86** [comment]
Maybe you can share code, screenshots, expected and actual results?
